distros/spring/src/main/java/com/connexta/transformation/service/TransformController.java

```diff
@@ -36,7 +45,7 @@
  */
 @RestController
 @CrossOrigin(origins = "*")
-public class TransformController implements TransformApi {
+public class TransformController {
```

Comment:
✏️ Unfortunately, for our version interceptor to work, we will need to continue implementing an interface defined as part of an artifact generated from the API repo. Could we split the `TransformApi` interface manually in that repo? This would also ensure that we don't modify these controllers structure unless the Api repo is released again.

❗️ We might want to actually have different controller classes for each endpoint instead of combining them in the same class.

Reply:
I was a little hesitant to extract the delete method into its own class because it is only 2 lines at this point. We may need to have a discussion with the other developers to see the best path forward for the endpoints as well as to all get on the same page.

Reply:
Having those in separate classes should ease maintenance and merge conflicts as it evolves.